tests/setup

```diff
@@ -144,8 +158,15 @@ setup_db() {
     if [[ -f lrl ]]; then
         cat lrl >> ${LRL}
     fi
-    if [[ -f lrl.options ]]; then
-        # use sed to substitute in test specific variables into lrl
+
+    # use sed to substitute in test specific variables into lrl
+    if [[ $dbrank -eq 3 && -f lrl_tertiary.options ]]; then
+        cat lrl_tertiary.options | sed "s#\${TESTDIR}#${TESTDIR}#" >> ${LRL}
+    elif [[ $dbrank -eq 2 && -f lrl_secondary.options ]]; then
+        cat lrl_secondary.options | sed "s#\${TESTDIR}#${TESTDIR}#" >> ${LRL}
+    elif [[ -f lrl.options ]]; then
+        # If a secondary or tertiary db doesn't have an explicit lrl file
+        # then use the basic lrl file.
         cat lrl.options | sed "s#\${TESTDIR}#${TESTDIR}#" >> ${LRL}
     fi
```

Comment:
If `lrl_secondary.options` doesn't exist and the database is a secondary database or if `lrl_tertiary.options` doesn't exist and the database is a tertiary database, then the database will use `lrl.options`, if it exists.